src/tiledb/cloud/utilities/_common.py

```diff
@@ -195,6 +196,74 @@ def run_dag(
     _print_logs(graph, debug=debug)
 
 
+def submit_taskgraph(
+    graph: dag.DAG,
+    *,
+    wait: bool = True,
+    update_sec: int = 10,
+) -> Mapping[str, str]:
+    """
+    Submit a taskgraph and optionally wait for completion.
+
+    :param graph: taskgraph to submit
+    :param wait: wait for the taskgraph to complete, defaults to True
+    :param update_sec: interval in seconds to update the progress bar, defaults to 10
+    :return: dictionary with status and graph_id
+    """
+
+    # Try to import tqdm, if not available, stub it out
+    try:
+        from tqdm import tqdm
+    except ImportError:
+
+        class tqdm:
+            def __init__(self, *args, **kwargs):
+                print("Please install tqdm to display a progress timer.")
+
+            def set_description_str(self, *args, **kwargs):
+                pass
+
+    graph.compute()
+
+    # Display a link to the log in TileDB, if available
+    if graph.server_graph_uuid:
+        print(
```

Comment:
Let's opt in to printing. I don't think a library function should print by default. I can't think of anything in Python's standard lib that prints except `print()`.

Like `if verbose and graph.server_graph_uuid:`